earthpy/plot.py

```diff
@@ -409,7 +409,7 @@ def hist(
     title=None,
     xlabel="",
     ylabel="",
-    **kwargs,
+    hist_range=[],
```

Comment:
This is the correct PR design @nkorinek as a side note...  `range` is a tuple in ax.hist. so you should call it range and then accept a tuple as the input value.